src/Skedify.test.js

```diff
@@ -590,6 +590,32 @@ describe('API', () => {
       ).toMatchSnapshot()
     })
 
+    it('should be possible to disable certain actions (update) on a resource', () => {
```

Comment:
Great test 👍 but do have a small remark on this (just for the record, doesn't need to be in scope of this issue); your test description is pretty generic (great) but the implementation is pretty specific for insights/cumulio (also great, but not for this test).

What I would suggest for the generic test; create a mock that explicitly disables 'update' action and verify that.

The specific test for insights/cumulio could be added to another test file (since this one is getting pretty large) where all current expectations of insights are verified.

Again, shouldn't really be in scope, as this change will have some setup implications and probably should be taken up in another issue. But wanted to mention it still (or get shut down if you disagree / see it differently 😃  )

Reply:
I prefer a big test file since this uses the public API of the SDK so that we don't use internals. And that brings me to the next point of using a mock. I don't want to touch internals so therefore I used an actual implementation. The example just uses an insights as an example but it can be anything here.

Reply:
If we want to support mocks from the public API that would also mean that we need a mechanism for injecting custom resource definitions so I am not entirely sure how we could / should do that. And even if we want that 🤔

Reply:
Let's discuss this some more, but yeah out of scope for now